CHANGELOG.md

```diff
@@ -20,26 +20,47 @@ argument or resolve the type ambiguity by casting to the intended type.
 **Bugfixes**
 * #4619 Improve handling enum columns in compressed hypertables
 * #4673 Fix now() constification for VIEWs
-* #4676 Fix a deadlock when decompressing chunks and performing SELECTs
 * #4681 Fix compression_chunk_size primary key
-* #4685 Improve chunk exclusion for space dimensions
 * #4696 Report warning when enabling compression on hypertable
+* #4745 Fix FK constraint violation error while insert into hypertable which references partitioned table
+* #4756 Improve compression job IO performance
+
+**Thanks**
+* @jvanns for reporting hypertable FK reference to vanilla PostgreSQL partitioned table doesn't seem to work
+
+## 2.8.1 (2022-10-04)
+
+This release is a patch release. We recommend that you upgrade at the
+next available opportunity.
+
+**Bugfixes**
+* #4454 Keep locks after reading job status
+* #4658 Fix error when querying a compressed hypertable with compress_segmentby on an enum column
+* #4671 Fix a possible error while flushing the COPY data
+* #4675 Fix bad TupleTableSlot drop
+* #4676 Fix a deadlock when decompressing chunks and performing SELECTs
+* #4685 Fix chunk exclusion for space partitions in SELECT FOR UPDATE queries
+* #4694 Change parameter names of cagg_migrate procedure
+* #4698 Do not use row-by-row fetcher for parameterized plans
+* #4711 Remove support for procedures as custom checks
+* #4712 Fix assertion failure in constify_now
+* #4713 Fix Continuous Aggregate migration policies
 * #4720 Fix chunk exclusion for prepared statements and dst changes
+* #4726 Fix gapfill function signature
 * #4737 Fix join on time column of compressed chunk
-* #4738 Fix the assorted epoll_ctl() errors that could occur with COPY into a distributed hypertable
+* #4738 Fix error when waiting for remote COPY to finish
 * #4739 Fix continuous aggregate migrate check constraint
-* #4745 Fix FK constraint violation error while insert into hypertable which references partitioned table
-* #4756 Improve compression job IO performance
 * #4760 Fix segfault when INNER JOINing hypertables
-* #4735 Allow manual index creation for CAggs
+* #4767 Allow manual index creation for CAggs
```

Comment:
```suggestion
* #4767 Fix permission issues on index creation for CAggs
```

I guess the current explanation is not very good because seems we're now allowing index creation on CAggs and actually it is a feature already released in 2.8.0 and this PR just fix a permission issue in this new feature.

Reply:
👍  Is changed in the current version of the PR